.gitignore

```diff
@@ -6,7 +6,9 @@
 !/cmake/*.cmake
 *~
 /test/benchmark_test
+/test/benchmark_test.exe
```

Comment:
Specifically adding the built executables makes sense for Unix platforms that don't have an executable extensions, but no need to duplicate this if there's an extension, just add a catchall.